### PR TITLE
Bugfix/proofs to the 1.0.0 documentation

### PR DESCRIPTION
**Pull request by stenskjaer**
From the commit messages:
 
Proofs to critical.md: 
A few of these corrections may be a matter of dispute, so I would
recommend looking through them.

Proofs to diplomatic.md:
Note that I am uncertain whether you can actually say "... mark
places *that* an ...". Here *where* seems more natural to me, but I may
be wrong. In that case, those hunks should be removed.

Generally: When merging these should/could be squashed into one commit.

### CODE REVIEW

File: docs/critical.md [hunk 1]
```diff
@@ -530,7 +530,7 @@ ad lumen solis
 
 This would make it easy to create the following apparatus entry:
 
-> nicticoracis *post* oculus A
+> nicticoracis *post* oculus *B*
```
Comment (jeffreycwitt) [inline docs/critical.md]: Why change B to *B* while changing below the *A* to A. Shouldn't this be "B", without italics.

File: docs/critical.md [hunk 2]
```diff
@@ -1459,17 +1461,17 @@ The tradition is unambiguous, but the editor deems it useless and prints an emen
 ``` xml
 Utrum fides
 <app>
-  <lem type="conjecture-corrected">sit</lem>
+  <lem type="conjecture-corrected"><corr>sit</corr></lem>
```
Comment (jeffreycwitt) [inline docs/critical.md]: good catch!

File: docs/critical.md [hunk 3]
```diff
@@ -1743,11 +1755,11 @@ A `cit` element that has a `quote` as well as a `bibl` and `note`:
 
 ### Description
 
-A quote, following the TEI guidelines, is intended to identify anything that is considered a quotation of another source or test. The value of quotation is most often rendered within quotation marks, i.e. "This is a quote".
+A quote, following the TEI guidelines, is intended to identify anything that is considered a quotation of another source or text. The value of quotation is most often rendered within quotation marks, i.e. "This is a quote".
 
 ### Rules
 
-1. `quote` **MAY** stand alone, i.e. it **MAY** not be a child of a `cit` element. In such a case it is generally expect that it would not appear in the *apparatus fonitum*.
+1. `quote` **MAY** stand alone, i.e. it **MAY** not be a child of a `cit` element. In such a case it is generally expected that it would not appear in the *apparatus fonitum*.
```
Comment (jeffreycwitt) [inline docs/critical.md]: fonitum should be "fontium". My bad.

File: docs/diplomatic.md [hunk 4]
```diff
@@ -325,7 +325,8 @@ The individual editor would usually not be responsible for maintaining the `revi
 ### Rules
 
 1. `front` **MUST** take a `div` with the `xml:id="starts-on"`.
-  * The content of this div should only be `pb`, `cb`, and `lb` elements indicating the page, column, and line on which the text begins in the different witnesses.
+  * The content of this div should only be `pb`, `cb`, and `lb` elements indicating the page, column, and line on which the text begins in the different witnesses. For the encoding of `pb` and `cb`
+ see [the relevant section below](#milestones).
```
Comment (jeffreycwitt) [inline docs/diplomatic.md]: for th encoding of `pb`, `cb` and `lb` see...

File: docs/intro-to-lbp-schema.md [hunk 5]
```diff
@@ -13,7 +13,7 @@ Editors:
...
+The [Textual Encoding Initiative](http://www.tei-c.org/) is—for many good reasons—considered the gold standard for textual editing. Despite holding such a privileged position, it is occasionally criticized for failing to make possible what many people hope to find in it: the silver bullet for achieving interoperability. But this criticism is misplaced as TEI has never claimed to provide a standard that by itself will ensure global interoperability. Rather TEI is better understood as first step towards interoperability: a tool box of diverse elements highly pertinent to a wide variety of textual projects. This tool box helps create a first level of uniformity by ensuring that diverse and scattered groups can adopt common nomenclature for so called 'low hanging fruit'. For example, TEI helps ensure that we all use `<p>` for paragraph rather than some groups using `<para>` and other groups using `<paragraph>`.
```
Comment (jeffreycwitt) [inline docs/intro-to-lbp-schema.md]: "as A first step towards interoperability" (phrase is missing an "a")

**Comment by jeffreycwitt** [comment]
Looks good. I noticed that you changed all sigla to italics. That's fine by me as along as we're consistent. I noticed a few "A" that were not changed to italic. (Actually changed from italic, to non-italic). If you don't mind correcting these and as well as the above corrections, then I pull this in.

**Comment by stenskjaer** [comment]
The italics is a complicated case. I switched back and forth, and then unfortunately ended somwhere in the middle (it wasn't consistent before either). Actually I had an evil plan about changing all to non-italic (and made a long commit message about it for a separate commit), as that's what I prefer. But then I had already made all (supposedly) italic and decided to keep it at that. But if you prefer non-italic, I would happily oblige. Otherwise I will make sure that they are all *actually* italic rather than just *almost* all.

**Comment by stenskjaer** [comment]
I have added the suggested changes.

**Pull request merged**